RecoEgamma/EgammaElectronProducers/plugins/alpaka/PixelMatchingAlgo.h

```diff
@@ -0,0 +1,23 @@
+#ifndef RecoEgamma_EgammaElectronProducers_plugins_alpaka_PixelMatchingAlgo_h
+#define RecoEgamma_EgammaElectronProducers_plugins_alpaka_PixelMatchingAlgo_h
+
+
+#include "DataFormats/PortableTestObjects/interface/alpaka/TestDeviceCollection.h"
+#include "DataFormats/EgammaReco/interface/alpaka/EleSeedDeviceCollection.h"
+#include "DataFormats/EgammaReco/interface/alpaka/SuperclusterDeviceCollection.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+#include <DataFormats/EgammaReco/interface/alpaka/Phys3DVector.h>
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
+
+  class PixelMatchingAlgo {
+  public:
+    void printEleSeeds(Queue& queue, reco::EleSeedDeviceCollection& collection) const;
+    void printSCs(Queue& queue, reco::SuperclusterDeviceCollection& collection) const;
```

Comment:
The `collection` arguments could be `const`
```suggestion
    void printEleSeeds(Queue& queue, const reco::EleSeedDeviceCollection& collection) const;
    void printSCs(Queue& queue, const reco::SuperclusterDeviceCollection& collection) const;
```